CRM/Eck/Page/Entity/TabHeader.php

```diff
@@ -94,7 +94,7 @@ public static function process(&$page): ?array {
     $module = _afform_angular_module_name($afform_name);
     $tabs['edit'] = [
       'title' => E::ts('Edit'),
-      'template' => 'CRM/Eck/Page/Entity/Edit.tpl',
+      'template' => 'afform/InlineAfform.tpl',
       'module' => $module,
       'directive' => _afform_angular_module_name($afform_name, 'dash'),
     ] + $default;
```

Comment:
@jensschuppe  This needs one more change to make it work:
```suggestion
    ] + $default;
    
    $page->assign('afformOptions', ['Eck_' . $entityType['name'] => $entityID]);
```